zbus_macros/src/proxy.rs

```diff
@@ -720,9 +720,10 @@ fn gen_proxy_property(
                 let receive = format_ident!("receive_{}_changed", method_name);
                 let gen_doc = format!(
                     "Create a stream for the `{property_name}` property changes. \
-                This is a convenient wrapper around [`{proxy_name}::receive_property_changed`]."
+                     This is a convenient wrapper around [`{proxy_name}::receive_property_changed`]."
```

Comment:
Is this change really relevant to the commit?

Reply:
Revert in f511f8fd0a50a81b7682ff091ac6323642207fdc.